models/preles/R/write.config.PRELES.R

```diff
@@ -23,17 +40,44 @@
 
 write.config.PRELES<- function(defaults, trait.values, settings, run.id){
   
-  # find out where to write run/ouput
+  ### Define PARAMETERS
```

Comment:
Similarly, it's unclear to me why you're parsing the parameters into this odd cmdFlags format when PRELES isn't reading them from the command line but rather just reading the file in. If runPRELES.jobsh is just reading a file, why not write it out as something quick and simple that doesn't require any parsing or restructuring (e.g. RData, csv)